code/modules/tgui/tgui.dm

```diff
@@ -127,6 +127,7 @@
  **/
 /datum/tgui/proc/close()
 	user << browse(null, "window=[window_id]") // Close the window.
+	src_object.ui_close()

```

Comment:
the hell is this

Reply:
it closes src_object' associated ui, duh

come to think if it i don't think ui objects ever GC